docs/_sources/about_reco.rst.txt

```diff
@@ -107,3 +107,18 @@ Improving the highest-ranked recommendations has a more important effect than im
 
 .. math::
     NDCG@k = \frac{1}{\left | A \right |} \sum_{i=1}^{\left | A \right |} \frac {\sum_{r=1}^{\left | P_i \right |} \frac{rel(P_{i,r})}{log_2(r+1)}}{\sum_{r=1}^{\left | A_i \right |} \frac{1}{log_2(r+1)}}
+
+Diversity Recommender Metrics
+-----------------------------
+Diversity recommender metrics evaluate the quality of recommendations in terms of the diversity of recommended items.
+
+Inter-List Diversity
+^^^^^^^^^^^^^^^^^^^^
+Inter-List Diversity@k measures the inter-list diversity of the recommendations when only k recommendations are
+made to the user. It measures how user's lists of recommendations are different from each other.
```

Comment:
Can we add something about it is range? I assume something in [0..1]? Then, what's better? Does a higher value mean it is more diverse? 

Some comments about how to interpret the result would be good the have. In the past, I don't know if we have done that for other metrics, but we can start with this one.